sharkfin/population.py

```diff
@@ -737,10 +737,10 @@ def assign_solution(self, agent):
 
         # assign solution before simulating
         # get master solution
-        pop_solution = self.pop.solution.solution_database
+        pop_solution = self.solution.solution_database
 
         # get solution for agent subgroup
-        functions = pop_solution.loc[agent.CRRA, agent.DiscFac]
+        functions = pop_solution.loc[agent.DiscFac, agent.CRRA]
```

Comment:
I swear I've changed this back and forth in order, and every time it seems the underlying dictionary's MultiIndex has the opposite order, so I'm not sure if order matters, but I also wonder if because these are sometimes long decimal floats that creates an issue.

Reply:
I think I've located the cause of the problem and will make a comment on that line.